### ISSUE TITLE
Optimise ARIA live region

### ISSUE THREAD

**Issue by jmuheim** [NothingAG/adg-components#42]
While the announcements to the screen reader used to work very well in the proof of concept, they do not work as fine in the current implementation. Especially in Firefox, we have some problems.

I experimented quite a lot to find solid ways to target different browsers, which would enhance the experience for certain screen readers. I had some insights, but I didn't really succeed yet. I asked on StackOverflow whether somebody knows a good solution:

https://stackoverflow.com/questions/72071457/what-is-the-best-way-to-announce-some-text-to-screen-readers-ie-role-alert

At the time being, @rudigier's suggestion to simply append a visually hidden `role="alert"` element to the DOM whenever an announcement is made, might be the most pragmatic way to go. ~~If you feel like trying that, @rudigier, please give it a go. It's not super important though, as it's not crucial for Tuesday (we can fix and improve it at a later stage).~~

UPDATE: The current implementation works pretty well in JAWS+Chrome, JAWS+FF, NVDA+Chrome, and VoiceOver/iOS. Actually, only NVDA+FF doesn't announce the live region's content at all (this is strange, as the `role="alert"` is clearly there, but there might be some special circumstances we are not yet aware of). Being one of the less relevant combinations (9.7% in desktop, see https://webaim.org/projects/screenreadersurvey9/), let's not put too much effort into this at the time being.

For the records: I'm playing around with a more robust approach to solve this (https://codepen.io/jmuheim/pen/qBxBqdE), but I'm not completely there yet.

**Comment by jmuheim** [comment]
The live region plays just fine at the time being: only NVDA+FF don't work yet, but they do not belong to the major combos anymore. Closing this.
